issue_processing/jira_issue_processing.py

```diff
@@ -330,6 +427,91 @@ def print_to_disk(issues, results_folder):
     csv_writer.write_to_csv(output_file, lines)
 
 
+def print_to_disk_bugs(issues, results_folder):
+    """Sorts of bug issues and prints them to file 'bugs-jira.list' in result folder
+
+    :param issues: the issues to sort of bugs
+    :param results_folder: the folder where to place 'bugs-jira.list' output file
+    """
+
+    # construct path to output file
+    output_file = os.path.join(results_folder, "bugs-jira.list")
+    log.info("Dumping output in file '{}'...".format(output_file))
+
+    # construct lines of output
+    lines = []
+    for issue in issues:
+        log.info("Current issue '{}'".format(issue['externalId']))
+
+        # only writes issues with type bug and their comments in the output file
+        if issue['type'] == "Bug":
+            lines.append((
+                issue['externalId'],
+                issue['state'],
+                issue['creationDate'],
+                issue['resolveDate'],
+                False,  ## Value of is.pull.request
+                issue['author']['name'],
+                issue['author']['email'],
+                issue['creationDate'],
+                issue['references'],
+                "open",  ## event.name
+                issue['resolution'],
+                issue['components']
```

Comment:
How does the CSV writer stores the components here? Comma-separated, as embedded JSON, ...?

Reply:
The components (as well as the references) get written in a list separated by a comma. Furthermore they get written as an Unicode-String.
This is agreed with Barbara but if you have an better idea I can easily change it.

Reply:
Okay, I just wanted to check whether Python behaves correctly here so that we can parse the data item later.